plugins/woocommerce/changelog/add-38285-add-filter-to-free-extensions-rest-endpoint

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add a filter to free extensions REST endpoint
```

Comment:
Same comment here as below for the filter comment. This isn't very useful to somebody glancing over the changelog what the change entry means. Please consider the audience for the change entry. What does the filter do?